templates/azure/terraform/example/hcl_properties.erb

```diff
@@ -12,6 +12,9 @@
 <%=       prop[0].ljust(prop_alignment) -%> = "<%= prop[1] -%>"
 <%      elsif prop[1].is_a?(Integer) || prop[1].is_a?(TrueClass) || prop[1].is_a?(FalseClass) -%>
 <%=       prop[0].ljust(prop_alignment) -%> = <%= lines(prop[1].to_s) -%>
+<%      elsif prop[1].is_a?(Array) && (prop[1].length == 0 || prop[1][0].is_a?(String))
+          res_arr = prop[1].collect{|elem| "\""+elem.to_s+"\""} -%>
```

Comment:
Can you try the following code whether it works or not?

```suggestion
          res_arr = prop[1].map{|elem| "\"#{elem}\""} -%>
```

Reply:
It works, replaced now.